assets/section-image-banner.css

```diff
@@ -321,3 +271,23 @@
 .banner__heading {
   margin-bottom: 0;
 }
+
+.banner__buttons {
+  display: inline-flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  max-width: 45rem;
+}
+
+.banner--text-align-right .banner__buttons--multiple { 
+  justify-content: flex-end;
+}
+
+.banner--text-align-center .banner__buttons--multiple > * { 
+  flex-grow: 1;
+  min-width: 22rem;
+}
```

Comment:
Should we have this ? Im curious as to what designers think. Basically it mean that if you set the text alignment to be centered on desktop, it will make the buttons full width whatever the value of `mobile text alignment` is. 
[Screenshot](https://screenshot.click/21-11-8880v-38q8q.png)

If we don't we could have this: 
- [left aligned with a long button label](https://screenshot.click/21-11-dm4gz-etsve.png)
- [center aligned with a long button label](https://screenshot.click/21-11-7fhdq-6jr8g.png)
- [right aligned with a long button label](https://screenshot.click/21-11-987pj-8o8d8.png)

Or at least only when it's set to `center` but for the `mobile text alignment` not desktop.